Userland/Utilities/wc.cpp

```diff
@@ -57,6 +57,21 @@ static ErrorOr<Count> get_count(StringView file_specifier)
         return count;
     }
 
+    if (only_bytes) {
+        struct stat file_info;
+        if (fstat(maybe_file.value()->fd(), &file_info) == -1) {
+            warnln("wc: unable to stat {}", file_specifier.is_empty() ? "stdin"sv : file_specifier);
+            count.exists = false;
+            return count;
+        }
+
+        if (S_ISREG(file_info.st_mode) || S_ISLNK(file_info.st_mode) || S_ISDIR(file_info.st_mode)) {
+            count.name = file_specifier;
+            count.bytes = file_info.st_size;
```

Comment:
:thinking: this is not _always_ correct, you'd have to at least open it and `seek(SEEK_END)` to get the right size.
```
❯ stat /proc/thread-self/auxv 
  File: /proc/thread-self/auxv
  Size: 0         	Blocks: 0          IO Block: 1024   regular empty file
  # blah blah

❯ wc -c /proc/thread-self/auxv 
368 /proc/thread-self/auxv
```

Reply:
> 🤷 the syscall gives you the right value, the kernel just doesn't populate a file in procfs just because you want to stat it.

Well… if the file isn’t populated then 0 *is* the correct value right?

Reply:
If you cat a file and it's 50 bytes long, but wc says it's 0, that sounds confusing and awkward.

Maybe as a workaround, do a full read if the stat reported size is 0?

Reply:
I agree it’s confusing, but I’d also argue that `wc` would end up being more correct. I don’t really like the idea of doing a (potentially slow) full read if the stat gives 0 though, so perhaps seeking to the end is the best way?

Reply:
Yeah I meant seeking not reading, I shouldn't phone-post when I've just woken up. :laughing:

Reply:
Pushed a new version using `seek()` and `tell()`.  Does this look better?